garden/src/components/form/garden/create/FormFields.tsx

```diff
@@ -346,9 +273,17 @@ const Step3 = () => {
         name="authors"
         render={({ field }) => (
           <FormItem>
-            <FormLabel>Authors *</FormLabel>
+            <FormLabel className="font-bold">Authors</FormLabel>
             <FormControl>
-              <MultipleSelector {...field} placeholder="Add authors" creatable />
+              <MultipleSelector
+                {...field}
+                placeholder="Add authors"
+                creatable
+                onChange={(value) => {
+                  field.onChange(value.map((v: any) => v.value));
+                }}
+                value={field.value.map((v: any) => ({ value: v, label: v }))}
```

Comment:
Seems like we need this mapping boilerplate in all the places we have array-valued fields? Looks like the right move, just a little surprising that the form library doesn't handle it more natively

Reply:
This is the best way I've found-- it's definitely possible there's a better way, I'm still finding new ways to do stuff